configure.ac

```diff
@@ -740,7 +736,6 @@ case "$enable_lastlog" in
   yes|check)
     BACKUP_LIBS=$LIBS
     LIBS="$LIBS -lutil"
-    AC_CHECK_FUNCS([logwtmp])
```

Comment:
$ac_cv_func_logwtmp is still used to define COND_BUILD_PAM_LASTLOG

Reply:
It's back in.